hybrasyl/World.cs

```diff
@@ -1652,6 +1652,49 @@ where searchTerm.IsMatch(amap.Key)
                         }
                     }
                         break;
+
+                    case "/rollchar":
+                    {
+                        // /rollchar <class> <level>
+                        // Allows you to "roll a new character" of the desired Class and Level, to see his resulting HP and MP.
+
+                        string errorMessage = "Command format is: /rollchar <class> <level>";
+                        byte level = 1;
+
+                        if (args.Length != 3)
+                        {
+                            user.SendMessage(errorMessage, 0x1);
+                        }
+                        else if (!Enum.IsDefined(typeof(Class), args[1]))
+                        {
+                            user.SendMessage("Invalid class. " + errorMessage, 0x1);
+
+                        }
+                        else if (!byte.TryParse(args[2], out level) || level < 1 || level > 99)

```

Comment:
FYI maxlevel elsewhere in hybrasyl is 255, since level is represented as a uint8. Just something to remember, doesn't really matter here.